sorc/link_workflow.sh

```diff
@@ -77,12 +77,24 @@ esac
 source "${top_dir}/versions/fix.ver"
 
 LINK="ln -fs"
-SLINK="ln -fs"
```

Comment:
Not sure about removing `SLINK`. `LINK` gets changed to be a copy for NCO, but some things they still allow to be symlinks. Changing those all to copy will substantially increase the size of the installation (and time it takes this script to complete).

Reply:
yeah, I understand.  Not sure what I was thinking.  reverted and added clarity